pkgs/top-level/all-packages.nix

```diff
@@ -2162,7 +2162,9 @@ with pkgs;
 
   ### APPLICATIONS/EMULATORS/BSNES
 
-  ares = callPackage ../applications/emulators/bsnes/ares { };
+  ares = darwin.apple_sdk_11_0.callPackage ../applications/emulators/bsnes/ares {
```

Comment:
Is this hack still needed?

Reply:
Even on aarch64-darwin, without setting `NIX_CFLAGS_COMPILE`, I had something like

```
make: *** [../nall/GNUmakefile:244: obj/ares-n64-rdp.o] Error 1
make: *** Waiting for unfinished jobs....
In file included from ../thirdparty/mame/mame/video/n64.cpp:28:
In file included from ../thirdparty/mame/mame/includes/n64.h:9:
../thirdparty/mame/mame/video/n64.h:132:7: error: aligned deallocation function of type 'void (void *, std::align_val_t) noexcept' is only available on macOS 10.14 or newer
class n64_rdp : public poly_manager<uint32_t, rdp_poly_state, 8>
      ^
../thirdparty/mame/mame/video/n64.cpp:3169:10: note: in implicit destructor for 'n64_rdp' first required here
n64_rdp::n64_rdp(n64_state &state, uint32_t* rdram, uint32_t* dmem) : poly_manager<uint32_t, rdp_poly_state, 8>(state.machine())
```

So I think it only builds on SDK 10.14+.